### PR TITLE
Erg #17 product teplate

### PR DESCRIPTION
**Pull request by derpdead**

### CODE REVIEW

File: components/Template/TemplateDesigner/LayoutElement.vue [hunk 1]
```diff
@@ -276,18 +282,12 @@ export default {
             this.newHeight = height;
         },
         initElementStyleForResizeState() {
-            this.$el.style.position = 'absolute';
             this.$el.style.width = `${this.startWidth}px`;
             this.$el.style.height = `${this.startHeight}px`;
-            this.$el.style.border = '2px solid #00bc87';
-            this.$el.style.zIndex = '5';
         },
         resetElementStyleForEndResizeInteraction() {
```
Comment (bleto) [inline components/Template/TemplateDesigner/LayoutElement.vue]: Better is use CSS class with property:
`width: 'unset;
height: 'unset''`

File: components/TemplateGrid/TemplateDesigner/TemplateGridGhostItem.vue [hunk 2]
```diff
@@ -100,30 +85,28 @@ export default {
...
-            }
-        },
-        removeGhostElementIfExist() {
             if (typeof this.draggedElement === 'object') {
-                removeGhostElementFromDraggableLayer();
+                draggedElementWidth = this.draggedElement.width;
+                draggedElementHeight = this.draggedElement.height;
             }
+
+            const elementsGap = 16;
```
Comment (bleto) [inline components/TemplateGrid/TemplateDesigner/TemplateGridGhostItem.vue]: Maybe better is set this variable as data state. You can use this as globally settings, and in the future move this to settings file.

File: components/Template/TemplateDesigner/LayoutElement.vue (hunk 1 above)
Comment (derpdead) [reply components/Template/TemplateDesigner/LayoutElement.vue]: CSS class won't working - inline styles will overwrite it

File: components/TemplateGrid/TemplateDesigner/TemplateGridGhostItem.vue (hunk 2 above)
Comment (derpdead) [reply components/TemplateGrid/TemplateDesigner/TemplateGridGhostItem.vue]: I would rather to keep it in method scope

**Pull request merged**